File: push_inbox_test/push_inbox_test/ViewController.m
//
//  ViewController.m
//  push_inbox_test
//
//  Created by Fatma Selin Hangişi on 12.05.2020.
//  Copyright © 2020 Webinstats. All rights reserved.
//

#import "ViewController.h"
@import iOS_wis;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    Webinstats* wiso = [[Webinstats alloc] init:@"//wis.webinstats.com/" :@"__COMPANY_ID__" :@"0"];
    NSMutableDictionary *map = [[NSMutableDictionary alloc] init];
    [map setObject:@"Home" forKey:@"p"];
    [map setObject:@"123" forKey:@"cuid"];
    [map setObject:@"1" forKey:@"_enable_push"];
    [map setObject:@"abc" forKey:@"wistest"];
    [wiso executeWithView:self localmap:map];
    // Do any additional setup after loading the view.
}


@end

File: DemoExample/DemoExample/ViewController.swift
//
//  ViewController.swift
//  DemoExample
//
//  Created by Fatma Selin Hangişi on 09/10/2017.
//  Copyright © 2017 Fatma Selin Hangişi. All rights reserved.
//

import UIKit
import iOS_wis

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let wiso = Webinstats("//demoapb.webinstats.com/","1525","0")
        var map = [String:String]()
        map["p"] = "Home Page"
        map["_enable_push"] = "1"
        wiso.execute(view: self, localmap: map)
        
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

